package.json

```diff
@@ -24,6 +24,7 @@
     "@leafygreen-ui/text-input": "1.1.1",
     "@leafygreen-ui/toggle": "3.0.1",
     "@leafygreen-ui/typography": "1.0.1",
+    "@mongodb-dev-prod/trend-charts-ui": "^0.2.3",
```

Comment:
we should remove the `^` so we can load a previous version of trend-charts-ui and redeploy if needed by only making changes to spruce instead of spruce and trend-charts-ui

Reply:
This will coerce lockstep releases with spruce, right? Any patch update to the trend charts ui will have us coming here to update this line. If you want to load and redeploy a previous version by only changing spruce, it seems like this would be the line to edit at that point. On our end it would be convenient to just iterate major/minor/patch versions with some care, and this UI to consume updates as appropriate, e.g. fixed major/minor version and latest patch.

Reply:
@SupaJoon @khelif96 Any thoughts on this? We'd prefer to not do lockstep releases so that we're not having to do two separate pr's whenever we make a minor change on the trend charts. If this is a hard requirement, that's fine though

Reply:
It's not a hard requirement for us to exclude the caret and I am for excluding the caret if it can benefit your workflow. 
Something to consider if we exclude the caret is that there isn't an easy way to tell which version of trend-charts-ui is deployed in spruce by looking at the build/dist files (you can see the output in the `build` folder by running `npm run build` in spruce). The most convenient way to understand which version of `trend-charts-ui` is deployed in Spruce would be to see what was available in the trend-charts-ui npm module at the time of the most recent spruce deploy.

Reply:
That should be fine. We can do `~` instead if that makes you more comfortable so that we're only automatically updating on patch updates. That way we don't have to bother yall with bug fixes or small changes but are marking the times when we're making larger changes.

Reply:
@zamj sounds good to me!